src/stories/Library/event-list-item/event-list-item.scss

```diff
@@ -12,8 +12,12 @@
   row-gap: $s-md;
   max-width: $layout__max-width;
 
-  > svg {
-    display: none;
+  // Extending display none, as mixin show-svg-on-hover
+  // is only working up to __small breakpoint
+  @include media-query__medium("max-width") {
+    > svg {
+      display: none;
```

Comment:
I will write this down and do it as part of the layout css adjustments coming in shortly instead of this pr. This way i can also refactor existing use cases.